hack/port-forward.sh

```diff
@@ -1,67 +1,6 @@
 #!/usr/bin/env bash
 set -eu -o pipefail
 
-pf() {
-  set -eu -o pipefail
-  resource=$1
-  port=$2
-  dest_port=${3:-"$port"}
-  ./hack/free-port.sh $port
-  echo "port-forward $resource $port"
-  kubectl -n argo port-forward "svc/$resource" "$port:$dest_port" &
-	until lsof -i ":$port" > /dev/null ; do sleep 1 ; done
-}
+go install github.com/kitproj/kubeauto@v0.0.7
```

Comment:
Nice! I made my own attempt at getting rid of `kubectl port-forward` a little while ago: https://github.com/argoproj/argo-workflows/compare/main...MasonM:argo-workflows:no-port-forward

The idea is to change all services to be of type `LoadBalancer`. With k3s, that means [ServiceLB will automatically forward the port to the host](https://docs.k3s.io/networking/networking-services?_highlight=loadb#service-load-balancer). This works, but it kinda couples the dev environment to k3s, since some distributions (e.g. kind) don't come with a `LoadBalancer` out-of-the-box. It looks like kubeauto doesn't have that problem.